docs/book/src/cronjob-tutorial/testdata/project/internal/webhook/v1/webhook_suite_test.go

```diff
@@ -29,8 +29,7 @@ import (
 	. "github.com/onsi/ginkgo/v2"
 	. "github.com/onsi/gomega"
 
-	admissionv1 "k8s.io/api/admission/v1"
-	apimachineryruntime "k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/client-go/kubernetes/scheme"
```

Comment:
HI @mateusoliveira43 

Thanks for suggesting this! I’ve been thinking about it, and I believe we should stick to the current setup for a few reasons:

It gives users granular control over adding only the API groups they need for their test cases, keeping things lightweight and precise. Not that it is not a bug. It is done in this one by design. Users can add ANY required scheme. 
- Using `client-go/kubernetes/scheme` adds all API groups by default, which increases memory usage and makes debugging schema issues harder.
**- The current approach is more streamlined for custom resources, avoiding unnecessary complexity or conflicts.**


Overall, I think the flexibility of the existing setup is better suited for varied test scenarios. 
Let me know your thoughts!

Best,

Reply:
OH, I think now I understand. 
The change proposed in https://github.com/kubernetes-sigs/kubebuilder/pull/4449 is to make it be as we have it setup for controllers now. 

Would be that @mateusoliveira43 ?

Reply:
yes, both for consistency (same setup as controllers test) and for fixing the bug with conversion webhooks